ios/Classes/MobileScanner.swift

```diff
@@ -434,16 +461,30 @@ public class MobileScanner: NSObject, AVCaptureVideoDataOutputSampleBufferDelega
     /// Analyze a single image
     func analyzeImage(image: UIImage, position: AVCaptureDevice.Position,
                       barcodeScannerOptions: BarcodeScannerOptions?, callback: @escaping BarcodeScanningCallback) {
-        let image = VisionImage(image: image)
-        image.orientation = imageOrientation(
+        var uiImage = image
```

Comment:
```suggestion
let uiImage: UiImage

if (invertImage) {
  uiImage = self.invertImage(image: image)
} else {
  uiImage = image
}
```

Can you also use a `let` here?